packages/eslint-plugin/rules/no-direct-date.test.js

```diff
@@ -36,7 +36,7 @@ const ruleTester = new RuleTester( {
 ruleTester.run( 'no-direct-date', rule, {
 	valid: [
 		'const date = select( CORE_USER ).getReferenceDate();',
-		'const createTime = new Date( Date.now() - DAY_IN_SECONDS * 1.5 * 1000 ).toISOString();',
+		'const createTime = new Date( DAY_IN_SECONDS * 1.5 * 1000, 20, 3 ).toISOString();',
```

Comment:
This instance isn't exactly using the constructor properly. The arguments are `year, monthIndex, day` which don't quite align with the usage here.

Reply:
Ah, true, I was just using bogus values but I'll have them reflect real-world usage 👍🏻